src/decisionengine_modules/glideinwms/resource_dist_plugins.py

```diff
@@ -1,7 +1,10 @@
 import structlog
 import pandas
 
-logger = structlog.getLogger()
+from decisionengine.framework.modules.logging_configDict import CHANNELLOGGERNAME
+
+logger = structlog.getLogger(CHANNELLOGGERNAME)
+logger = logger.bind(module=__name__.split(".")[-1], channel="")
```

Comment:
Here to get the module attribute to stick should be something like
```python
logger = structlog.getLogger(f"{CHANNELLOGGERNAME}.glideinwms.resource_dist_plugins")
logger = logger.bind(module=__name__.split(".")[-1])
```
The logger name could also be only `f"{CHANNELLOGGERNAME}.resource_dist_plugins"`, but I thought the other one is more clear. Alternatively `f"{CHANNELLOGGERNAME}.{__name__}"` which could be used all the time in the module level getLogger() calls.